benchmark/README.md

```diff
@@ -0,0 +1,39 @@
+# How to Run the Benchmark tests
+
+## Prerequisites
+
+1.Install ROS 2 from binary
+
+You can download the latest binary package of ROS2 from [here](http://ci.ros2.org/view/packaging/) and follow the instructions to setup the environment.
+
+* [Linux](https://github.com/ros2/ros2/wiki/Linux-Install-Binary)
+* [macOS](https://github.com/ros2/ros2/wiki/OSX-Install-Binary)
+* [Windows](https://github.com/ros2/ros2/wiki/Windows-Install-Binary)
+
+2.Install Node.js
```

Comment:
Better to indicate the `Node.js` version, suggest to use latest `Node.js` LTS version always.

Reply:
I will add to point out the Node.js version as latest LTS